addons/website_forum/static/src/js/website_forum.js

```diff
@@ -178,6 +178,14 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             });
         });
 
+        this.$('.o_wforum_question, .o_wforum_answer, .o_wforum_post_comment, .o_wforum_last_activity').toArray()
+            .forEach((post) => {
+                const createDateTimeISO = $(post).data('createDate').split('.')[0].replace(' ', 'T') + 'Z';
```

Comment:
do others convert our dates manually to get ISO? Doesn't luxon provide some kind of format option?

Reply:
Changing this part to avoid converting the datetime manually.